src/universal/modules/userDashboard/components/UserActionListEmpty/UserActionListEmpty.js

```diff
@@ -0,0 +1,87 @@
+import React, {PropTypes} from 'react';
+import look, {StyleSheet} from 'react-look';
+import theme from 'universal/styles/theme';
+import {ib} from 'universal/styles/helpers';
+import ui from 'universal/styles/ui';
+import {Type} from 'universal/components';
+import FontAwesome from 'react-fontawesome';
+
+const iconStyle = {
+  ...ib,
+  color: theme.palette.mid,
+  marginRight: '.25rem',
+  textAlign: 'center',
+  width: '1rem'
+};
+
+const padding = '.5rem';
+
+const UserActionListEmpty = (props) => {
+  const {styles} = UserActionListEmpty;
+  const {onCreateNewAction} = props;
+  return (
+    <div className={styles.root}>
+      <div className={styles.header}>
+        <FontAwesome name="plus-circle" onClick={onCreateNewAction} style={iconStyle} />
+        <Type bold display="inlineBlock" lineHeight="1.125rem" onClick={onCreateNewAction} scale="s3" width="auto">
+          Add New Action
+        </Type>
+      </div>
+      <div className={styles.body}>
+        <FontAwesome className={styles.hintIcon} name="lightbulb-o" style={iconStyle} />
+        <div style={{overflow: 'hidden'}}>
+          <Type family="serif" italic lineHeight="1.25rem" scale="s2">
+            Actions are smaller tasks not tracked on the team dashboard;{' '}
+            only you can see them after they are created.{' '}
+            They are great for things you can complete in under a day or two.
```

Comment:
I've been considering omitting this last sentence "They are great..." but let's see how it looks/tests.